plugins/lease_time/plugin.go

```diff
@@ -0,0 +1,54 @@
+// Copyright 2018-present the CoreDHCP Authors. All rights reserved
+// This source code is licensed under the MIT license found in the
+// LICENSE file in the root directory of this source tree.
+
+package leasetime
+
+import (
+	"errors"
+	"time"
+
+	"github.com/coredhcp/coredhcp/handler"
+	"github.com/coredhcp/coredhcp/logger"
+	"github.com/coredhcp/coredhcp/plugins"
+	"github.com/insomniacslk/dhcp/dhcpv4"
+)
+
+func init() {
+	plugins.RegisterPlugin("lease_time", nil, setupLeaseTime4)
+}
+
+var (
+	log         = logger.GetLogger("plugins/lease_time")
+	v4LeaseTime time.Duration
+)
+
+// Handler4 handles DHCPv4 packets for the lease_time plugin.
+func Handler4(req, resp *dhcpv4.DHCPv4) (*dhcpv4.DHCPv4, bool) {
+	if req.OpCode != dhcpv4.OpcodeBootRequest {
+		log.Warningf("not a BootRequest, ignoring")
```

Comment:
This should be an Infof at the max, probably a Debugf

Reply:
I removed this logging completely...not sure anyone would even want to debug that.